front-end/src/app/transactions/transaction-group-b/transaction-group-b.component.ts

```diff
@@ -28,7 +29,37 @@ export class TransactionGroupBComponent extends TransactionTypeBaseComponent imp
     'memo_code',
     'memo_text_input',
   ];
-  override contactTypeOptions: PrimeOptions = LabelUtils.getPrimeOptions(ContactTypeLabels).filter((option) =>
-    [ContactTypes.INDIVIDUAL, ContactTypes.ORGANIZATION, ContactTypes.COMMITTEE].includes(option.code as ContactTypes)
-  );
+
+  override ngOnInit(): void {
+    if (this.transaction?.transactionType?.scheduleId === 'B') {
```

Comment:
I see is this what you were talking about pulling out, but held back because of scope creep?

Reply:
I was talking about moving the TransactionTypes and TransactionGroups out of the Transaction file and into the TransactonType file